<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PostDetail</class>
 <widget class="QWidget" name="PostDetail">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>帖子详情</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="titleLabel">
     <property name="text">
      <string>帖子标题</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="contentTextEdit">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="repliesLabel">
     <property name="text">
      <string>回复列表:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="refreshRepliesButton">
     <property name="text">
      <string>刷新回复</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="repliesTextEdit">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="replyLayout">
     <item>
      <widget class="QLineEdit" name="replyLineEdit">
       <property name="placeholderText">
        <string>输入回复内容...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="replyButton">
       <property name="text">
        <string>回复</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
